components/ModalTemplateSwitchApplicationNetwork.vue

```diff
@@ -3,16 +3,17 @@ const { closeModal, modal } = useModal()
 const { formatMessage } = useIntl()
 
 const handleChangeNetwork = async () => {
-  const { selectedChainId } = storeToRefs(useAppStore())
   const { getNetworkByChainId } = useAppStore()
   const { disconnect } = useBaseProvider()
 
-  selectedChainId.value = modal?.data?.chainId
   await navigateTo({
```

Comment:
You can just pass external: true to reload this, no?

Reply:
Ah yes, I forgot about this